src/ai/backend/manager/registry.py

```diff
@@ -1872,6 +1896,8 @@ def get_image_conf(kernel: KernelRow) -> ImageConfig:
                             "idle_timeout": int(idle_timeout),
                             "mounts": [item.to_json() for item in scheduled_session.vfolder_mounts],
                             "environ": {
+                                **_pytorch_distributed_environ,
+                                **_tf_distributed_environ,
```

Comment:
Do you always apply the environment for pytorch and tf?

Reply:
Per-image basis would be better. Thanks!